[#14] [Documentation] Document infrastructure setup (#39)
Nihisil/nimble-devops-ic-infrastructure: .github/wiki/Infrastructure-Documentation.md

This document contains a description of the requisite software and hardware needed to run infrastructure for an [example app](https://github.com/Nihisil/nimble-devops-ic-app).

Infrastructure diagram is available [here](Infrastructure Documentation)

## Required hardware

All parts are hosted on AWS and they rely heavily on AWS services. It would require complete restructuring to migrate them to a different cloud provider.

RDS instance type:
- `db.t4g.medium`

ElastiCache instance type:
- `cache.t3.small`

ECS Fargate instance type:
- Refer to `ecs` variable description below in this documentation

## Required software

> The modifications to the infrastructure are being brought about through Terraform Cloud.

Reviewer: this sentence is hard to understand (at least for me). How about rephrasing them, such as
```suggestion
Terraform Cloud is used to make modifications to the infrastructure.
```

Author: Thank you. It was changed here: https://github.com/Nihisil/nimble-devops-ic-infrastructure/pull/39/commits/5bdd7dd649767f7078b7dfacc0824d0b18b2789e
